### ISSUE TITLE
Unable to activate "Accessibility" for RethinkDNS on GrapheneOS (55a works, 55c does not)

### ISSUE THREAD

**Issue by mcctuxic** [celzero/rethink-app#1260]
Hi,

First of all: GREAT APP !!! :+1:   Thank you :)

To be able to distinguish between active and inactive apps, one has to activate "Accessibility" in the Android settings for the Rethink-App 

That works fine with version 55a (F-Droid) and is "deactivated for security reasons" (that is: grayed out - message is from a popup of the OS) when tried for version 55c (github).

This makes a vital feature unusable.

Found on:
Google Pixel 6a, GrapheneOS (patch 20240308, latest version, OTA installation)

**Comment by mcctuxic** [comment]
I had an exception in the same area ("Accessibiliy") this morning with version 55a. It was the first and single event.
(sorry, no native speaker here).

**Comment by mcctuxic** [comment]
I have found a solution - kind of...
One can enable Accessibility services for RethinkDNS 55c on GrapheneOS according this post:

https://stackoverflow.com/questions/72217216/android-13-enable-accessibility-service-for-apk-which-installed-not-from-googl

(yes, the 'e' in the last word is missing from the URL)

But I cannot explain why 55c needs this step and 55a does not.

May be it has something to do with the signature?
55a was installed form F-Droid and 55c was downloaded and installed from github...
But that is pure guesswork on my side.

One is for sure: GREAT APP! :)

**Comment by OrkoGrayskull** [comment]
There is a fix for this: https://support.google.com/android/answer/12623953?hl=en

**Comment by Domy81** [comment]
> There is a fix for this: https://support.google.com/android/answer/12623953?hl=en
> 
>     1. On your Android device, open the Settings app.
> 
>     2. Tap Apps.
> 
>     3. Tap the app that you want to turn on a restricted setting for. (RethinkDNS)
> 
>     4. Tap More (the dots) and then Allow restricted settings.
> 
>     5. Follow the on-screen instructions.

It doesn't work on Android 15.
Any solution?

**Comment by Domy81** [comment]
Can you fix it on Android 15, please?
